homeassistant/components/xbox/media_player.py

```diff
@@ -111,10 +111,9 @@ def state(self):
     @property
     def supported_features(self):
         """Flag media player features that are supported."""
-        active_support = SUPPORT_XBOX
         if self.state not in [STATE_PLAYING, STATE_PAUSED]:
-            active_support &= ~SUPPORT_NEXT_TRACK & ~SUPPORT_PREVIOUS_TRACK
-        return active_support
+            return SUPPORT_XBOX & ~SUPPORT_NEXT_TRACK & ~SUPPORT_PREVIOUS_TRACK
```

Comment:
Unrelated to this PR but `supported_features` should not change dynamically.

Reply:
Oh okay, good to know.

Reply:
Well sadly for media players they currently are ok to change dynamically. That needs to be fixed first by splitting media_features out of supported features